src/http.rs

```diff
@@ -123,6 +124,17 @@ where
                 })
                 .context(CallRequestSnafu)?;
 
+            let (parts, body) = response.into_parts();
+            let mut buf: Vec<u8> = Vec::new();
+            let body = body.map_data(move |data| {
+                buf.extend(data);
+                if buf.len() > 20 {
+                    println!("Data buf has reached len 20: {:?}", std::str::from_utf8(&buf));
+                }
+                bytes::Bytes::new()
```

Comment:
This needs to return the incoming `data`, otherwise all of the response data will be dropped.